.github/workflows/e2e-ci.yml

```
@@ -103,13 +130,6 @@ jobs:
         run: |
           export CI_OCI_CERTS_DIR="$(git rev-parse --show-toplevel)/FleetCI-RootCA"
 
-          # 1. Run test cases not needing infra
-          ginkgo --github-output --label-filter='!infra-setup' e2e/single-cluster e2e/keep-resources e2e/drift
-
-          # 2. Run tests for metrics
-          ginkgo --github-output e2e/metrics
-          SHARD=shard1 ginkgo --github-output e2e/metrics
-
           # 3. Run tests requiring only the git server
```

On comments 3. to 6.: we should either reset the numbering to start with 1 (or 0 :upside_down_face: ) or remove that numbering altogether. I think the comments still help readability here.
```suggestion
          # Run tests requiring only the git server
```